### PR TITLE
Apr 1292 organisation high risk

### PR DESCRIPTION
**Pull request by MakCom**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _logger = new Mock<ILogger<GatewayOrganisationChecksOrchestrator>>();
+            _orchestrator = new GatewayOrganisationChecksOrchestrator(_applyApiClient.Object, _logger.Object);
+        }
+
+        [TestCase("Company and charity", "Eva Training and Consultancy")]
+        [TestCase(null, "Eva Training and Consultancy")]
+        [TestCase("Company and charity", null)]
+        [TestCase(null, null)]
+        public void Check_orchestrator_builds_with_organisation_risk(string organisatioType, string tradingName)
```
Comment (David-B-Read) [inline src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs]: Typo in variable name

Comment (MakCom) [reply src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs]: typo is fixed now.

File: src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/OrganisationChecks/OrganisationRiskTests.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                Status = SectionReviewStatus.Pass,
+                SourcesCheckedOn = DateTime.Now,
+                ErrorMessages = new List<ValidationErrorDetail>()
+            };
+
+            _applyApiClient.Setup(x => x.SubmitGatewayPageAnswer(applicationId, pageId, vm.Status, username, comment));
+
+            var result = _controller.EvaluateOrganisationRiskPage(vm).Result;
+
+            _orchestrator.Verify(x => x.GetOrganisationRiskViewModel(new GetOrganisationRiskRequest(applicationId, username)), Times.Never());
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/OrganisationChecks/OrganisationRiskTests.cs]: You'll need a verification that the response has been saved.

File: src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _logger = new Mock<ILogger<GatewayOrganisationChecksOrchestrator>>();
+            _orchestrator = new GatewayOrganisationChecksOrchestrator(_applyApiClient.Object, _logger.Object);
+        }
+
+        [TestCase("Company and charity", "Eva Training and Consultancy")]
+        [TestCase(null, "Eva Training and Consultancy")]
+        [TestCase("Company and charity", null)]
+        [TestCase(null, null)]
+        public void Check_orchestrator_builds_with_organisation_risk(string organisationType, string tradingName)
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs]: Can you remove the technical detail from the test name?

Comment (MakCom) [reply src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs]: Renaming is done.

File: src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/OrganisationChecks/OrganisationRiskTests.cs (hunk 2 above)
Comment (MakCom) [reply src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/OrganisationChecks/OrganisationRiskTests.cs]: It's done

File: src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs (hunk 1 above)
Comment (MakCom) [reply src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs]: It's fixed

**Pull request merged**